charts/castai-agent/values.yaml

```diff
@@ -18,27 +18,28 @@ image:
 nameOverride: ""
 fullnameOverride: castai-agent
 
-# Specifies whether a namespace should be created
+# createNamespace -- Specifies whether a namespace should be created
```

Comment:
We should probably stay consistent and either add period at the end of each value description or none of them